### DISCUSSION TITLE
وارد نشدن به پنل پس از نصب با لینک

### DISCUSSION THREAD

**Discussion by koichfidel102** [hiddify/Hiddify-Manager#550] [Q&A سوالات رایج]
سلام پس از نصب وقتی لینک برای ورود میدهد مشکل ارور کانکشن به وجود میاد تو کروم 
![Uploading Screenshot_20230331_194319_Chrome.jpg…]()

**Comment by esmaeilifardi** [comment]
rvf5
خطایی رخ داده است!!
(sqlite3.InterfaceError) Error binding parameter 1 - probably unsupported type. [SQL: SELECT bool_config.child_id AS bool_config_child_id, bool_config."key" AS bool_config_key, bool_config.value AS bool_config_value FROM bool_config WHERE bool_config."key" = ? AND bool_config.child_id = ? LIMIT ? OFFSET ?] [parameters: ('telegram_enable', <built-in function 3>, 1, 0)] (Background on this error at: https://sqlalche.me/e/14/rvf5)


این بار دومه روی اوبونتو نسخه آخر و سرور خام آپدیت شده نصبش میکنم خطا میده 
دفعه پیشم خطای nginx برای auto cdn و سرتیفیکیت های nginx میداد
کلا گیرو گور زیاد داره به نظرم نسخه اصلی نصب بشه خیلی بهتره

**Reply by Lorenzo19877** [reply]
برا منم همینه دوبار امتحان کردم از اول نصب کردم مشکل چیه پیدا کردین؟

**Reply by veniz666** [reply]
vase man aslan bala neiyad

**Comment by esmaeilifardi** [comment]
سلام مهندس
نه والا بیخیالش شدم
bash <(curl -Ls
https://raw.githubusercontent.com/FranzKafkaYu/x-ui/master/install_en.sh)
اینو نصب کردن

**Comment by behcom** [comment]
بعد نصب لینک رو کپی میکنم وارد پنل بشم ارور میده
